src/java/org/apache/cassandra/db/partitions/TriePartitionUpdate.java

```diff
@@ -311,6 +326,17 @@ public int dataSize()
         return dataSize;
     }
 
+    /**
+     * The size of the data contained in this update.
+     *
+     * @return the size of the data contained in this update.
+     */
+    @Override
+    public long unsharedHeapSize()
+    {
+        return dataSize;
+    }
```

Comment:
For Review:
`accumulatedDataSize` and `unsharedHeapSize` are coming from 5.0 before `PartitionUpdate` became an interface. The implementation has moved to `BTreePartitionUpdate` and `TriePartitionUpdate`.